css/main.css

```diff
@@ -1,3 +1,225 @@
 /* Add your CSS here */
+*{
 
+}
+
+body{
+    /* display: grid; */
+    /* grid-template-columns: ; */
+    /* min-width: auto; */
+    margin-top: 5rem;
+    font-family: "poppins", sans serif;
+  }
+
+.topnav{
+    display: flex;
+    flex-direction: row;
+     
+}
+
+#myLinks a {
+  text-decoration: none;
+  color: black;
+  float: right;
+  width:7em;
+  margin:10px auto;
+  font-size: 15px;
+}
+
+
+.container{
+  width:100%;
+  display: grid;
+  grid-template-areas: " left right";
```

Comment:
Here's a fun idea. You could  remove the gap rule and instead define _empty_  columns in CSS grid like this:

`grid-template-areas: ". card . intro .";`

Then you could try setting rules about how those columns are sized something like this:

`grid-template-columns: 1fr minmax(auto, 375px) 20px 3fr 1fr`

Have a play around.